ccvm_simulators/ccvmplotlib/problem_metadata/boxqp_metadata.py

```diff
@@ -83,69 +81,26 @@ def ingest_metadata(self, metadata_filepath: str) -> None:
         self.__batch_size = self.__df["batch_size"][0]
         self.__problem_size_list = sorted(self.__df["problem_size"].unique().tolist())
 
-    def generate_arrays_of_TTS(
-        self, matching_df: pd.DataFrame, percent_gap: str, problem_size: int
-    ) -> list:
-        """A method to generate more data points that is used for plotting.
-
-        With a given result data in pandas.DataFrame, it generates best known
-        energy list, Time-To-Solve list, and success probability list based on
-        the optimal type and problem size inputs.
+    def generate_TTS_plot_data(
+        self,
+        method="advanced",
```

Comment:
The variable `method` is not used and it's not really needed. In all the calculations of TTS we should just use the "advanced" method. We can remove this variable.